src/ctl/src/lib.rs

```diff
@@ -49,6 +49,12 @@ enum Commands {
     Bench(BenchCommands),
     /// Commands for tracing the compute nodes
     Trace,
+    // TODO(yuhao): profile other nodes
+    /// Commands for profilng the compute nodes
+    Profile {
+        #[clap(short, long = "sleep")]
+        sleep: u64,
```

Comment:
Why name it `sleep`? 🤔 Suggest `sampling_time_ms`.

Reply:
linux perf use sleep

Reply:
Let's unify the namings for CLI args and RPC args.